components/application-broker/internal/broker/provision_test.go

```diff
@@ -710,6 +716,7 @@ func TestDoProvision(t *testing.T) {
 				mockServiceInstanceGetter,
 				fake.NewSimpleClientset().ApplicationconnectorV1alpha1(),
 				knative.NewClient(knCli, k8sCli),
+				istioversionedclientfake.NewSimpleClientset(),
```

Comment:
How do we validate the policy which got created?

Reply:
Still not validated. I suggest creating a ticket for proper testing. What do you think?
CC: @Abd4llA ?

Reply:
Now its tested.